tests/dbus/test_dbus_client.cpp

```diff
@@ -349,6 +349,59 @@ void CheckTelemetryData(ThreadApiDBus *aApi)
 #if OTBR_ENABLE_LINK_METRICS_TELEMETRY
     TEST_ASSERT(telemetryData.low_power_metrics().link_metrics_entries_size() >= 0);
 #endif
+#if OTBR_ENABLE_BORDER_AGENT
+    TEST_ASSERT(telemetryData.wpan_border_router().border_agent_info().epskc_state() ==
```

Comment:
better to create a reference pointing to border agent info so the code can be more concise

Reply:
Done